internal/automation/publish_release.go

```diff
@@ -36,5 +35,5 @@ func newPublishRunner(cfg *config.Config) *publishRunner {
 }
 
 func (r *publishRunner) run(ctx context.Context) error {
-	return runCommandFn(ctx, publishCmdName, r.projectID, r.push, false)
+	return runCommandFn(ctx, publishCmdName, r.projectID, false, false)
```

Comment:
This seems a bit weird. both commands call this `runCommandFn` and publish_release is forced to pass in a flag it does not use.

Reply:
See https://github.com/googleapis/librarian/pull/2844#discussion_r2519100011